### PR TITLE
Employer login

### PR DESCRIPTION
**Pull request by Tsaikkari**
Added employerLogin controller and router.

### CODE REVIEW

File: src/entities/Employer.postgres.ts [hunk 1]
```diff
@@ -39,4 +39,8 @@ export default class Employer extends BaseEntity {
     cascade: ['remove'],
   })
   jobPosts!: JobPost[]
+
+  static localLogin(email: string, password: string) {
+    return this.find({ where: { email: email, password: password } })
+  }
```
Comment (Wyfy0107) [inline src/entities/Employer.postgres.ts:45]: I don't think you're using this anywhere. You are using `findOne` already in the passport config. Can you remove it ?

**Review by Wyfy0107 — CHANGES_REQUESTED** [review]
💯 Looks good overall, however, missing test. Can you write test for this in **empoyer.test.ts**. Also open the pr to **feature-controller** branch

File: src/controllers/employerLogin.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  try {
+    await passport.authenticate('local', function (error, employer) {
+      if (!employer) {
+        next(new NotFoundError('Email or password not found', error))
+      }
+    })
```
Comment (Wyfy0107) [inline src/controllers/employerLogin.ts]: what about the case where `employer` exists ?

**Review by Wyfy0107 — CHANGES_REQUESTED** [review]

File: src/entities/Employer.postgres.ts (hunk 1 above)
Comment (Tsaikkari) [reply src/entities/Employer.postgres.ts:45]: I'm using this in the employerLogin file to connect passport strategy with controller. Shall I remove the file?

File: src/controllers/employerLogin.ts (hunk 2 above)
Comment (Tsaikkari) [reply src/controllers/employerLogin.ts]: success

File: src/routers/auth.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import express from 'express'
+
+import { localLogin } from '../controllers/employerLogin'
+
+const router = express.Router()
+
+router.post('/login/local', localLogin)
+
```
Comment (Nejo12) [inline src/routers/auth.ts:8]: 👍

**Review by Nejo12 — DISMISSED** [review]

**Pull request closed without merging**